### PR TITLE
Update README.md and CONTRIBUTING.md files

### LINKED ISSUE #2538: Add back GH Badges
We recently disabled several of them because they were not in good working order. This is a backlog issue to add them back when there is time. Might be a good opportunity to incorporate the new fancy badges from https://shields.io/category/build

Related to https://github.com/patternfly/patternfly-react/issues/2525

### LINKED ISSUE #2933: fix(documentation): Update React documentation
- Remove outdated information
- Remove redundant information from subproject READMES
- Add badges and custom build steps to each subproject README
- Add information about monorepo
- Update unmaintained generators

### PR DESCRIPTION
**Pull request by jessiehuff**
Noticed a lot of duplication and outdated information in the documentation, thought an update could help

Fixes #2933

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,48 @@
...
+export const MyComponent = props => (
+  <Button>This is an interactive PatternFly 4 styled button!</Button>
+);
+```
+
+### PatternFly 3
+
+`npm install --save patternfly-react`
 
-This project is governed by the Contributor Covenant version 1.4. All contributors and participants agree to abide by its terms. To report violations, send an email to patternfly@redhat.com.
```
Comment (tlabaj) [inline README.md]: Why did we remove this?

File: README.md [hunk 2]
```diff
@@ -1,26 +1,48 @@
...
+
+export const MyComponent = props => (
+  <Button>This is an interactive PatternFly 3 styled button!</Button>
+);
+```
 
 ### Contributing Guide
 
-If you are interested in contributing to PatternFly React, please see the [Contributing Guide](./CONTRIBUTING.md).
+If you are interested in contributing to PatternFly React, please see [Getting Started as a Contributor](https://github.com/patternfly/patternfly-react/wiki/Getting-Started-as-a-Contributor) and the [Contributing Guide](./CONTRIBUTING.md).
```
Comment (tlabaj) [inline README.md]: Can you remove reference to the wiki here.  Hard to find contribution guide from there.

File: packages/patternfly-3/patternfly-react/CONTRIBUTING.md [hunk 3]
```diff
@@ -15,95 +15,6 @@
...
-A link to the PatternFly 3 Storybook will be automatically generated and posted as a comment after the pull request build is complete.
-
-Once your pull request has been reviewed, if all conditions above have been met your pull request will be approved and merged.
-
-Please help in ensuring all relevant issues are closed and that any subsequent issues needed have been noted with this pull request.
-
 ## Additional Information
 
-See the PatternFly React Guide for full details on [Code Contribution Guidelines](https://github.com/patternfly/patternfly-react/blob/master/CONTRIBUTING.md#code-contribution-guidelines)
+See the PatternFly React Guide for full details on [Code Contribution Guidelines](../../CONTRIBUTING.md#code-contribution-guidelines)
```
Comment (tlabaj) [inline packages/patternfly-3/patternfly-react/CONTRIBUTING.md]: the link does not work... Why did you remove the contribution instructions from the guide?

File: packages/patternfly-4/README.md [hunk 4]
```diff
@@ -1,3 +1,51 @@
...
+
+These packages are internal to the project and for developers.
+
+### react-docs
+
+Gatsby project to demonstrate our components.
+
+### react-integration
+
+[Cypress](https://www.cypress.io/) suite with a Create React App demo project to test TypeScript types.
```
Comment (tlabaj) [inline packages/patternfly-4/README.md]: Can you remove "to test Typescript types".  The purpose of react-integration is to test our components in an integration environment similar to what a consumer would have in order to find API issues early.

File: packages/patternfly-4/react-charts/README.md [hunk 5]
```diff
@@ -1,33 +1,16 @@
...
-```
+Use your favorite package manager to install our package from npm: 
 
-or
+`yarn add @patternfly/react-charts`
 
-```
-npm install @patternfly/react-charts --save
-```
+`npm install @patternfly/react-charts --save`
```
Comment (tlabaj) [inline packages/patternfly-4/react-charts/README.md]: should there be an or before this line?

File: packages/patternfly-4/react-core/CONTRIBUTING.md [hunk 6]
```diff
@@ -7,102 +7,22 @@ section: patternfly-4
...
-7.  Push
-
-```text
-$ git push origin my-branch
-```
-
-8.  Create a pull request
-
-    - A link to the PatternFly 4 demo documentation will be automatically generated and posted as a comment after the pull request build is complete.
+1. Write the component implementation in `[Component].tsx`.
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/CONTRIBUTING.md]: these are not numbered correctly

File: packages/patternfly-4/react-core/CONTRIBUTING.md [hunk 7]
```diff
@@ -7,102 +7,22 @@ section: patternfly-4
...
-- When you select the option to generate a PatternFly 4 component, a structure resembling the following is generated
-  ```text
-  packages/patternfly-4/react-core/src/[type]/[ComponentName]/
-    index.js - Barrel File exporting public exports
-    ComponentName.js - Component Implementation
-    ComponentName.test.js - Component Tests
-    ComponentName.md - Component Docs
-  ```
-
-4.  Develop your component. After development is complete, ensure tests and lint standards pass.
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/CONTRIBUTING.md]: can you add these steps back in.  I think they are important for contributors.

File: packages/patternfly-3/patternfly-react/CONTRIBUTING.md [hunk 8]
```diff
@@ -15,95 +15,6 @@
 4. Add tests to `[ComponentName].test.js`. All new components must be tested.
 5. Update the generated storybook `[ComponentName].stories.js`. When your stories contain multiple files, put them in a subfolder named `Stories`. Stories should demonstrate as many different UI states for your component as possible. Use Storybook knobs to enable dynamic visualizations of your component's props. For PatternFly 4 components, provide associated examples for documentation in the examples directory for the component.
 
-## Code Contribution Guidelines
```
Comment (tlabaj) [inline packages/patternfly-3/patternfly-react/CONTRIBUTING.md]: Where are these steps now?

File: packages/patternfly-4/react-core/README.md [hunk 9]
```diff
@@ -1,45 +1,32 @@
...
-yarn add @patternfly/react-core
-```
-
-or
+`yarn add @patternfly/react-core`
 
-```
-npm install @patternfly/react-core --save
-```
+`npm install @patternfly/react-core --save`
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/README.md]: I think we need an or so someone does not do both.

File: packages/patternfly-4/react-core/README.md [hunk 10]
```diff
@@ -49,64 +36,18 @@ export default <Button variant="primary">Button</Button>;
...
+## Building
 
-Run to install all the dependencies
+To build just react-core, run `yarn run lerna run build --scope=@patternfly/react-core`.
 
-```sh
-yarn install && yarn build
-```
-
-### Running
```
Comment (tlabaj) [inline packages/patternfly-4/react-core/README.md]: How do they know how to run it?

File: packages/patternfly-4/react-table/README.md [hunk 11]
```diff
@@ -86,50 +68,11 @@ If you want to add this transformer as default cell (first, last, any,...) for e
...
-You can also find how each component is meant to be used from a design perspective on the [PatternFly 4 Core][patternfly-4] website.
-
-Note: All commands below assume you are on the root directory in this repository.
-
-### Install
-
-Run to install all the dependencies
-
-```sh
-yarn && yarn bootstrap && yarn build && yarn build:docs
```
Comment (tlabaj) [inline packages/patternfly-4/react-table/README.md]: Is this no longer needed?

File: packages/patternfly-4/react-table/README.md [hunk 12]
```diff
@@ -86,50 +68,11 @@ If you want to add this transformer as default cell (first, last, any,...) for e
...
 Notice: Any data provided as cell will be visible by default, no need to add `isVisible` if you want to change how data are displayed.
 
-### Documentation
-
-This project uses Gatsby. For an overview of the project structure please refer to the [Gatsby documentation - Building with Components](https://www.gatsbyjs.org/docs/building-with-components/).
-
-A comprehensive list of components and detailed usage of each can be found on the [PatternFly React Docs][docs] website
-You can also find how each component is meant to be used from a design perspective on the [PatternFly 4 Core][patternfly-4] website.
-
-Note: All commands below assume you are on the root directory in this repository.
```
Comment (tlabaj) [inline packages/patternfly-4/react-table/README.md]: We no longer have to be at the root?

File: packages/patternfly-4/react-topology/README.md [hunk 13]
```diff
@@ -1,21 +1,9 @@
 # @patternfly/react-topology
 
-This package provides Topology View components based on [PatternFly 4][patternfly-4]
-
-### Prerequisites
-
-#### Node Environment
-
-This project currently supports Node [Active LTS](https://github.com/nodejs/Release#release-schedule) releases. Please stay current with Node Active LTS when developing patternfly-react.
```
Comment (tlabaj) [inline packages/patternfly-4/react-topology/README.md]: is this no longer a prerequisite?

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -33,7 +33,7 @@ agree to abide by its terms. To report violations, send an email to [patternfly@
 
 ## Issues and Project Board
 
-We use issues to track work items, such as bug reports and feature requests. Issues can be found in the [issue tracker](https://github.com/patternfly/patternfly-react/issues) or [project board](https://github.com/patternfly/patternfly-react/projects/1). We use the project board to help visualize and manage status of an issue, and we use labels to help prioritize and identify issues.
+We use issues to track work items, such as bug reports and feature requests. Issues can be found in the [issue tracker](https://github.com/patternfly/patternfly-react/issues) or on our [Zenhub project board](https://github.com/patternfly/patternfly-react/issues#workspaces/pf4-workspace-5b2142ff9499cb7cdaf1e632/board?repos=84372347). The [Zenhub extension](https://www.zenhub.com/extension) is required to view the project board. We use the project board to help visualize and manage status of an issue, and we use labels to help prioritize and identify issues.
```
Comment (tlabaj) [inline CONTRIBUTING.md]: We can have zenhub links here.  Not everyone has it installed.

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -112,26 +112,11 @@ To create an issue for adding a new component to the repo, please observe the fo
         - For PatternFly 4 components, direction will be given as to which package it will go in.  Generally, we are recommending most extension components go in individual packages.  Several smaller common components may end up being grouped together.  In the future, a generator for extension component packages will be created.
     - Extension components do not undergo the same rigorous design or coding review process as core PatternFly components.  If enough members of the community find them useful, we will work to move them into our core PatternFly system by starting the design process for the idea.
 
-### Contributing Components
-
-Components that are ready to be contributed can be found on the project board in the backlog column. They are identified with the text “Component” in the issue title.
```
Comment (tlabaj) [inline CONTRIBUTING.md]: I think we still need something here about how to contribute a component.

File: CONTRIBUTING.md [hunk 16]
```diff
@@ -164,60 +149,26 @@ Currently the following generators are provided
...
 - Provide a [jest snapshot test](https://facebook.github.io/jest/docs/snapshot-testing.html) to ensure your UI markup does not change unexpectedly.
 - Ensure the component's rendered design and Storybook stories for PatternFly 3 or Documentation examples for PatternFly 4 meet [PatternFly design standard](https://github.com/patternfly/patternfly-design).
   **Note:** If your component does not yet have PatternFly design documentation, the PatternFly React design team will first confirm that the pattern passes the [PatternFly Decision Tree](https://github.com/patternfly/patternfly-design/blob/master/resources/decision-tree/PatternflyDecisionTree.pdf) and then start the process for generating design documentation.
-- Ensure the code is properly formatted and there are no linting errors. PatternFly React uses custom eslint configuration based on [Javascript Standard Style](https://standardjs.com/) and [Prettier](https://github.com/prettier/prettier) for code formatting. You can automatically format your code with `yarn prettier` and run the project's linter with `yarn lint`.
```
Comment (tlabaj) [inline CONTRIBUTING.md]: I think you should put this back and give updated instructions for the linter.

File: CONTRIBUTING.md [hunk 17]
```diff
@@ -164,60 +149,26 @@ Currently the following generators are provided
...
-- When destructuring or spreading expressions , use ...props as the variable name.
-- Exporting components from other libraries (without manipulating them)
-  to consumers is a common task, use:
-
-  ```js
-  # Badge/Badge.js
-  import { Badge } from 'react-bootstrap';
-  export default Badge;
-
-  # Badge/index.js
```
Comment (tlabaj) [inline CONTRIBUTING.md]: Maybe update instead of removing?

File: CONTRIBUTING.md [hunk 18]
```diff
@@ -164,60 +149,26 @@ Currently the following generators are provided
...
-  export { default as Badge } from './Badge';
-
-  # index.js
-  export * from './Badge';
-  ```
-
-  **Please see the [Getting Started Readme](./GETTING-STARTED.md) for additional information in getting started with building PatterFly 4 react components.**
```
Comment (tlabaj) [inline CONTRIBUTING.md]: How will folks easily find the readme?

File: CONTRIBUTING.md [hunk 19]
```diff
@@ -228,45 +179,19 @@ cd patternfly-react
...
 
-2.  Create a branch:
+2. Create a branch:
 
 ```text
-$ git checkout -b my-branch -t upstream/master
+$ git checkout -b my-branch upstream/master
 ```
 
-3. Generate your Component
```
Comment (tlabaj) [inline CONTRIBUTING.md]: Are we no longer supporting generator?  It is mentioned elsewhere in the docs.

File: GETTING-STARTED.md [hunk 20]
```diff
@@ -1,281 +0,0 @@
-# PatternFly React Getting Started
```
Comment (tlabaj) [inline GETTING-STARTED.md]: Where are we capturing some of this information that is still relevant?

File: README.md (hunk 1 above)
Comment (redallen) [reply README.md]: It's still at the bottom, the diff didn't pick it up the best.

File: README.md (hunk 2 above)
Comment (redallen) [reply README.md]: Sure, we can not add the wiki link.

File: packages/patternfly-3/patternfly-react/CONTRIBUTING.md (hunk 8 above)
Comment (redallen) [reply packages/patternfly-3/patternfly-react/CONTRIBUTING.md]: These steps are out of date and catered towards adding components (which is not done often). Most of them are how to use `git` (which isn't specific to our project), and the only thing left that's relevant is `yarn generate` for scaffolding new components, which we can document better elsewhere. `yarn test` is rather obvious, and `yarn commit` isn't required anymore since semantic commits aren't required.

File: packages/patternfly-3/patternfly-react/CONTRIBUTING.md (hunk 3 above)
Comment (redallen) [reply packages/patternfly-3/patternfly-react/CONTRIBUTING.md]: The link needs one more `../` since it's relative.

File: packages/patternfly-4/README.md (hunk 4 above)
Comment (redallen) [reply packages/patternfly-4/README.md]: Sure, we can state it's larger purpose.

File: packages/patternfly-4/react-charts/README.md (hunk 5 above)
Comment (redallen) [reply packages/patternfly-4/react-charts/README.md]: We should add back the `or`.

File: packages/patternfly-4/react-core/CONTRIBUTING.md (hunk 7 above)
Comment (redallen) [reply packages/patternfly-4/react-core/CONTRIBUTING.md]: See previous reasons for PF3's CONTRIBUTING.md

File: packages/patternfly-4/react-core/CONTRIBUTING.md (hunk 6 above)
Comment (redallen) [reply packages/patternfly-4/react-core/CONTRIBUTING.md]: MD formats them correctly when rendered.

File: packages/patternfly-4/react-core/README.md (hunk 9 above)
Comment (redallen) [reply packages/patternfly-4/react-core/README.md]: We should add back the `pr`.

File: packages/patternfly-4/react-core/README.md (hunk 10 above)
Comment (redallen) [reply packages/patternfly-4/react-core/README.md]: We don't "run" `react-core`. We only build it. We should add to the root README.md `yarn start:pf3` and `yarn start:pf4` runs the separate environments.

File: packages/patternfly-4/react-table/README.md (hunk 12 above)
Comment (redallen) [reply packages/patternfly-4/react-table/README.md]: React-table doesn't need Gatsby docs.

File: packages/patternfly-4/react-table/README.md (hunk 11 above)
Comment (redallen) [reply packages/patternfly-4/react-table/README.md]: No.

File: packages/patternfly-4/react-topology/README.md (hunk 13 above)
Comment (redallen) [reply packages/patternfly-4/react-topology/README.md]: No. All LTS releases work.

File: CONTRIBUTING.md [hunk 21]
```diff
@@ -164,60 +149,26 @@ Currently the following generators are provided
...
 - Ensure the component's rendered design and Storybook stories for PatternFly 3 or Documentation examples for PatternFly 4 meet [PatternFly design standard](https://github.com/patternfly/patternfly-design).
   **Note:** If your component does not yet have PatternFly design documentation, the PatternFly React design team will first confirm that the pattern passes the [PatternFly Decision Tree](https://github.com/patternfly/patternfly-design/blob/master/resources/decision-tree/PatternflyDecisionTree.pdf) and then start the process for generating design documentation.
-- Ensure the code is properly formatted and there are no linting errors. PatternFly React uses custom eslint configuration based on [Javascript Standard Style](https://standardjs.com/) and [Prettier](https://github.com/prettier/prettier) for code formatting. You can automatically format your code with `yarn prettier` and run the project's linter with `yarn lint`.
-
-### Code Consistency
-
-- All files and folders under your package's `src/components` should name with PascalCase except `index.js` files
```
[the remark this answers is not in the document]
Comment (redallen) [reply CONTRIBUTING.md]: Which ones? We no longer lint files consistently, see #2362 .

File: CONTRIBUTING.md (hunk 14 above)
Comment (redallen) [reply CONTRIBUTING.md]: We should have a tutorial for how to install Zenhub.

File: CONTRIBUTING.md (hunk 15 above)
Comment (redallen) [reply CONTRIBUTING.md]: I think most contributors already know to just copy+paste current components.

File: CONTRIBUTING.md (hunk 16 above)
Comment (redallen) [reply CONTRIBUTING.md]: Agreed, we should fix the linter.

File: CONTRIBUTING.md (hunk 18 above)
Comment (redallen) [reply CONTRIBUTING.md]: By looking in the correct `packages/patternfly-4` folder.

File: packages/patternfly-4/react-core/README.md (hunk 10 above)
Comment (tlabaj) [reply packages/patternfly-4/react-core/README.md]: Yeah, but we should probably have steps here someone interested in seeing the demo's for react-core, so they can easily start the demo without having to dig for instructions how.

File: CONTRIBUTING.md (hunk 17 above)
Comment (redallen) [reply CONTRIBUTING.md]: We link to the react-docs README now instead of duplicating the information twice.

File: CONTRIBUTING.md (hunk 19 above)
Comment (redallen) [reply CONTRIBUTING.md]: Code contribution does not require generating a component. We include information about Generators under `### Using Generators`.

File: GETTING-STARTED.md (hunk 20 above)
Comment (redallen) [reply GETTING-STARTED.md]: Which parts aren't covered?
